modules/commandhandler/index.js

```diff
@@ -0,0 +1,103 @@
+/* global JitsiMeetExternalAPI */
+import load from 'load-script2';
+import config from '../../config.js';
+
+/**
+ * The jitsi-meet api.
+ */
+let api;
+
+/**
+ * Handles JitsiMeetExternalAPI command.
+ *
+ * @param {string} type - command type
+ * @param {string} args - arguments for the command
+ * @returns {string} execution status message
+ */
+function handleCommand(type, args) {
+    if (api) {
```

Comment:
this code branches too much and it's hard to read. can you try to think of a way to make it simpler?